sixtyfps_runtime/corelib/Cargo.toml

```diff
@@ -51,6 +53,7 @@ unicode-segmentation = "1.8.0"
 # Note: the rgb version is extracted in ci.yaml for rustdoc builds
 rgb = "0.8.27"
 pin-project = "1"
+atomic-polyfill = { version = "0.1.5", optional = true }
```

Comment:
since this is not an optional dependency of vtable, we don't need to have it as an optional dependency